node/src/crypto/asymmetric_key.rs

```diff
@@ -282,7 +300,7 @@ mod tests {
         assert_eq!(Some(Ordering::Equal), low.partial_cmp(&low_copy));
     }
 
-    mod ed25519 {
+    mod test_ed25519 {
```

Comment:
This is already inside a module named `test` - let's avoid adding the unnecessary prefix here.

Reply:
The `secp256k1` module was colliding with the library of the same name, so I changed it to `test_secp256k1`.  I changed this one too for symmetry.